docs/Build.md

```diff
@@ -102,7 +102,7 @@ Webpack allows file specific loaders or utilities to be invoked as a part of the
 | Rule                        | Plugin/loader(s)                                                                                        |                                                                                                                                                                                                   Purpose |
 | --------------------------- | ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
 | `/(\.css\|.scss)$/`         | `style-loader` (dev only), `miniCssExtractPlugin.loader` (production only), `css-loader`, `sass-loader` | Handle scss/css loading/references. If dev mode, use `style-loader` for speed, else use `miniCssExtractPlugin.loader` (in combination with the `miniCssExtractPlugin` plugin) to produce/emit css file(s) |
-| `/(\.ts|tsx)$/`             | `ts-loader`                                                                                             |                                                                                                                                                               Transpiles TypeScript files into JavaScript |
+| `/(\.ts                     | tsx)\$/`                                                                                                |                                                                                                                                                                                               `ts-loader` | Transpiles TypeScript files into JavaScript |
```

Comment:
I saw the same in #65 , but this breaks the table formatting - I take it the linter did this?

Reply:
Also @pmuir - PR #65 will resolve both changes made in this PR. That being the case, is this PR needed?